### ISSUE TITLE
Ability to reset all preferences to defaults

### ISSUE THREAD

**Issue by lacymorrow** [snapcrunch/electron-preferences#97]
Would like a helper function to automatically remove any non-default properties and set all default properties to their default values.

**Comment by lacymorrow** [comment]
This is how I handle it in my app

```js
const resetPreferences = () => {

	const { defaults } = preferences
	for ( const [ key, value ] of Object.entries( defaults ) ) {

		preferences.value( key, value )

	}

}
```

**Comment by pvrobays** [comment]
Released in 2.7.0!

**Comment by lacymorrow** [comment]
hell yeah!
